app/views/transaction/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= render "shared/navbar", {text:"Add Transaction",path:category_transaction_index_path(params[:category_id])} %>
+<div class="container mt-3">
+  <%= form_with model: @transaction, url:category_transaction_index_path(params[:category_id]) do |f| %>
+    <div class="form-group mb-3">
+      <%= f.text_field :name,  class:"form-control", Placeholder:"Name"%>
+    </div>
+    <div class="form-group mb-3">
+      <%= f.number_field :amount, class:"form-control", placeholder:"Amount" %>
+    </div>
+    <%= f.submit "Add Transaction",class:"btn btn-primary btn-sm" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Please know that the new transaction form should have the following: 
  * name (mandatory)
  * amount (mandatory)
  * categories (mandatory at least one)
  
  What you're missing is the "categories" I know that you're adding the category automatically but you should have the option to select at least one.
  
  ![image](https://user-images.githubusercontent.com/20567503/209165608-346ce6ba-d102-483b-8a48-76312600b3f4.png)
  
  Kindly add this as well as it is required!